parquet-thrift/src/main/java/parquet/hadoop/thrift/ThriftReadSupport.java

```diff
@@ -73,6 +73,12 @@ public static void setRecordConverterClass(JobConf conf,
     conf.set(RECORD_CONVERTER_CLASS_KEY, klass.getName());
   }
 
+  public static void setRecordConverterClass(Configuration conf,
```

Comment:
By the way, this tripped me up -- should ScroogeInputFormat call this method automatically? It seems odd to use ScroogeInputFormat and then also have to call this method?